test/specs/commands/verifyMessage.js

```diff
@@ -0,0 +1,135 @@
+							given.aPublicKey,
+							() => {
+								Given(
+									'a signature "KjyhJ+/Peyv2KsjDsfWs9pl8q2K6n941Z9GI7cusvF3IF3+4jQOoaRzgM0j1abEhvKnno8Q79cBWOC81/4Q8CQ=="',
+									given.aSignature,
+									() => {
+										Given(
+											'an empty options object',
+											given.anEmptyOptionsObject,
+											() => {
+												When(
+													'the action is called with the options',
+													when.theActionIsCalledWithTheOptions,
+													() => {
+														Then(
+															'it should reject with validation error and message "No public key was provided."',
+															then.itShouldRejectWithValidationErrorAndMessage,
+														);
+													},
+												);
+												When(
+													'the action is called with the options and the public key',
+													when.theActionIsCalledWithTheOptionsAndThePublicKey,
+													() => {
+														Then(
+															'it should reject with validation error and message "No signature was provided."',
+															then.itShouldRejectWithValidationErrorAndMessage,
+														);
+													},
+												);
+												When(
+													'the action is called with the options, the public key and the signature',
+													when.theActionIsCalledWithTheOptionsThePublicKeyAndTheSignature,
+													() => {
+														Then(
+															'it should reject with validation error and message "No message was provided."',
+															then.itShouldRejectWithValidationErrorAndMessage,
+														);
+													},
+												);
+											},
+										);
+										Given(
+											'an options object with message set to "messageSource"',
+											given.anOptionsObjectWithMessageSetTo,
+											() => {
+												Given(
+													'an error "Unknown data source type." occurs retrieving the inputs from their sources',
+													given.anErrorOccursRetrievingTheInputsFromTheirSources,
+													() => {
+														When(
+															'the action is called with the options, the public key, the signature and the message',
+															when.theActionIsCalledWithTheOptionsThePublicKeyTheSignatureAndTheMessage,
+															() => {
+																Then(
+																	'it should reject with the error message',
+																	then.itShouldRejectWithTheErrorMessage,
+																);
+															},
+														);
+													},
+												);
+												When(
+													'the action is called with the options, the public key and the signature',
+													when.theActionIsCalledWithTheOptionsThePublicKeyAndTheSignature,
+													() => {
+														Then(
+															'it should resolve to the result of verifying the message',
+															then.itShouldResolveToTheResultOfVerifyingTheMessage,
+														);
+													},
+												);
+											},
+										);
+										Given('a message "Hello world"', given.aMessage, () => {
```

Comment:
I think it makes more sense to nest this under `Given('an empty options object')` rather than duplicating that step.